index.rst

```diff
@@ -1,42 +1,621 @@
-..
-  Content of technical report.
+:tocdepth: 1
 
-  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Notation and Conventions
+========================
 
-  Use the following syntax for sections:
+Given multiple overlapping images :math:`\{z_1({\bf r}), z_2({\bf r}), ...\}`, we designate by :math:`R({\bf r})` the set of image indices that overlap a point on the sky :math:`{\bf r}`.  We assume that all images have already been resampled to a common coordinate system and common photometric units.  Each image is related to the true sky :math:`f({\bf r})` via its point spread function (PSF) :math:`\phi_i({\bf r}, {\bf s})` by
 
-  Sections
-  ========
+.. math::
+  z_i({\bf r}) = \int \! d^2{\bf s} \phi_i({\bf r}, {\bf s}) \, f({\bf s})
+    + \mathrm{noise}
+  :label: eq:psf_definition_continuous
 
-  and
+It will be convenient to rewrite this integral as a sum, and hence define the PSF and sky as discrete quantities rather than continuous functions (note that :math:`{\bf r}` already only takes discrete values).  The true :math:`f` has power at arbitrarily high frequencies, so we cannot simply sample it on a grid.  But the PSF does not; we can always choose some grid with points :math:`{\bf t}` upon which we can sample the PSF and exactly reconstruct it with Sinc interpolation:
 
-  Subsections
-  -----------
+.. math::
```

Comment:
Perhaps introduce them as discrete the first time with explicit indices but then saw that we're going to supress the indices henceforth?